getting_started/intel_debug/introduction.rst

```diff
@@ -142,16 +142,24 @@ See :ref:`llext_modules` for technical details.
 
 SOF project releases for Intel platforms are either monolithic (only a single firmware binary) or modular (base firmware and external libraries).
 
-1.3.1. Modular SOF releases
----------------------------
+1.3.1. Monolithic SOF releases
+------------------------------
 
-See :ref:`loadable-libraries` for details about library support in general.
+The release contains:
+ - **sof-PLAT.ri** : The firmware binary
+ - **UUID.bin** : Mainly 3rd party libraries identified by UUID. If the library contains multiple modules then a UUID symlink must be provided for each one. Optional, only supported with IPC4 and on Meteor Lake and newer platforms.
```

Comment:
should this really still be called "monolithic?" To me "monolithic" is when you have no loadable modules at all, no run-time code loading. Do we need to document 3 types then? Monolithic, individual modules and bundled modules?

Reply:
I think it was a mistake to release the DRC as module in a tech demo style for LNL, but my point here is:
monolithic: basefw (+optional UUID libraries, if any)
modular: basefw + openmodules + debug (+optional UUID libraries, if any)

The item tells that the UUID.bin is only supported on MTL+, which is correct.

Reply:
@ujfalusi  Github ate my comment (or user error), but I did intend to comment at this as well. I get @lyakh 's concern here, were are in fact shipping a modular build in 2024.09 , so it is what it is. 

I do like the "Modular Release" name, so not sure how to name the "before" and "after" layouts. Maybe just call it what is, "SOF Modular Release Layout V2 (for PTL and newer)", and "SOF Modular Release Layout V1" (for LNL/ARL/MTL)". Not pretty but describes what it is.

Reply:
I consider the UUID based loading as monolithic+library. The LNL is a short sighted tech demo to show that we can load a single module (as library) to the firmware.

But technically MTL+ will be able to load modular release (basefw+openmodules+debug+UUID), it is just that sof-bin will only going top ship modular release for PTL+.

My distinction criteria is:
If there is basefw _and_ openmodules _and_/_or_ debug) in the firmware path then it is a modular release, if there is only the basefw in there, it is monolithic.

The kernel will not care about any of these platform names, it will try to see if these fixed named libraries are there and if they are, load them.

and the UUID based library loading is also supported from MTL+ So, on PTL also.

Monolithic release == single library in fw path
Modular release == multiple libraries in fw path

Note: basefw is a library.

Reply:
I agree with the name monolithic being a bit of a misnomer too. It hints at not supporting loadable modules.

I have a suggestion, how about legacy(for the old releases) and bespoke (for the multiple binaries version) releases?